website/docs/r/application_gateway.html.markdown

```diff
@@ -391,6 +391,8 @@ A `waf_configuration` block supports the following:
 
 * `rule_set_version` - (Required) The Version of the Rule Set used for this Web Application Firewall.
 
+* `disabled_rule_group` - (Optional) one or many `disabled_rule_group` blocks as defined below.
```

Comment:
We should keep this message consistent with the other attributes of a similar nature
```suggestion
* `disabled_rule_group` - (Optional) One or more `disabled_rule_group` blocks as defined below.
```